src/components/app/exporters/YnabExporter.vue

```diff
@@ -14,33 +14,94 @@
       label="Budget id"
       outlined
+      @change="changed = true"
+    />
+    <ynab-account-mapping-table
+      :account-numbers-to-ynab-account-ids="accountNumbersToYnabAccountIdsArray"
+      @mappingChanged="markChanged()"
+      @addAccountMapping="addAccountMapping()"
+      @deleteAccountMapping="deleteAccountMapping($event)"
     />
-    <v-btn
-      color="primary"
-      :disabled="!readyToSave.value"
-      @click="submit()"
-    >
-      Save
-    </v-btn>
+    <v-container>
+      <v-row>
+        <v-col cols="10">
+          <v-btn
+            color="primary"
+            :disabled="!readyToSave.value"
+            @click="submit()"
+          >
+            Save
+          </v-btn>
+        </v-col>
+        <v-col cols="2">
+          <v-btn
+            color="blue"
+            fab
+            dark
+            small
+            @click="addAccountMapping()"
+          >
+            <v-icon dark>
+              mdi-plus
+            </v-icon>
+          </v-btn>
+        </v-col>
+      </v-row>
+    </v-container>
   </v-form>
 </template>
 
 <script lang="ts">
 import Vue from 'vue';
 import { setupExporterConfigForm } from '@/components/app/exporters/exportersCommon';
 import { OutputVendorName } from '@/originalBudgetTrackingApp/commonTypes';
+import { computed } from '@vue/composition-api';
+import YnabAccountMappingTable from '@/components/app/exporters/YnabAccountMappingTable.vue';
+import { YnabConfig } from '@/originalBudgetTrackingApp/configManager/configManager';
 
 export default Vue.extend({
   name: 'YnabExporter',
 
+  components: { YnabAccountMappingTable },
+
   setup() {
+    const dataToReturn = setupExporterConfigForm(OutputVendorName.YNAB);
+    const ynabConfig = dataToReturn.exporter as YnabConfig;
+    const accountNumbersToYnabAccountIdsArray = computed(() => {
+      return Object.keys(ynabConfig.options.accountNumbersToYnabAccountIds).map((accountNumber) => ({
+        accountNumber,
+        ynabAccountId: ynabConfig.options.accountNumbersToYnabAccountIds[accountNumber]
+      }));
+    });
+
     return {
-      ...setupExporterConfigForm(OutputVendorName.YNAB)
+      ...dataToReturn,
+      accountNumbersToYnabAccountIdsArray,
+      submit: () => {
+        const updatedAccountNumbersToYnabAccountIds = {};
+        accountNumbersToYnabAccountIdsArray.value.forEach(({ accountNumber, ynabAccountId }) => {
+          updatedAccountNumbersToYnabAccountIds[accountNumber] = ynabAccountId;
+        });
+        ynabConfig.options.accountNumbersToYnabAccountIds = updatedAccountNumbersToYnabAccountIds;
+
+        dataToReturn.submit();
+      },
+      deleteAccountMapping: (accountNumber) => {
+        Vue.delete(ynabConfig.options.accountNumbersToYnabAccountIds, accountNumber);
+        dataToReturn.changed.value = true;
+      },
+      addAccountMapping: () => {
+        Vue.set(ynabConfig.options.accountNumbersToYnabAccountIds, '###', '###');
+      },
```

Comment:
```suggestion
      addAccountMapping: () => {
        Vue.set(ynabConfig.options.accountNumbersToYnabAccountIds, '', '');
      },
```

I prefer that way, you can disagree with me.

For now it's OK to work with `object`, but I think, at least in the form, it should be an array of `key-value`, with rule to not duplicate the keys. Because as it now, you have a problem with adding more then one items. You have to add an item, fill it, then add another one... But it is not critical for now.